src/app/visual/autocomplete/autocomplete-visual.component.html

```diff
@@ -51,6 +51,14 @@ <h2>
     </form>
   </div>
 
+  <button
+    id="autocomplete-reactive-disable-button"
+    class="sky-btn sky-btn-primary"
```

Comment:
Check your alphabetical ordering here.

Reply:
Fixed. Also fixed a couple other places in this file that were wrong historically.